mmfunctions

```diff
@@ -1182,7 +1182,11 @@ _get_audio_mapping(){
     if [[ ( "${AUDIOMAP}" = "L" || "${AUDIOMAP}" = "R" ) && "${FIRSTMONO}" = true ]] ; then
         _report -wt "Channel mapping was requested, but first track is mono. Request ignored."
     fi
-    if [[ "${AUDIOMAP}" = "L" && "${FIRSTMONO}" != true ]] ; then
+    if [[ "${AUDIOMAP}" = "M" ]] ; then
+        _report -wt "Only use the first track."
+        _add_audio_filter -p "channelmap=0:mono"
```

Comment:
This line would only use the first channel of the first track. Not sure that this is what is intended.

Reply:
Yes, this is what I intended. I have a number of files that have 2 mono tracks, but the second track actually doesn't have any audio, so in order to make access copies, I want to only use the first channel of the first track.